src/site/content/en/blog/using-conversion-measurement/index.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Using the Event Conversion Measurement API
+subhead: Must-dos and tips to use the Event Conversion Measurement API.
+authors:
+  - maudn
+hero: hero.jpg
+date: 2020-11-12
+updated: 2020-11-12
```

Comment:
```suggestion
#updated: 2020-11-12
```
Nitpick: we can omit this until we actually do update the article sometime in the future